packages/cozy-sharing/src/state.js

```diff
@@ -370,11 +370,18 @@ const getSharedDocIds = doc =>
     ? getSharingDocIds(doc)
     : getPermissionDocIds(doc)
 
-export const getSharingDocIds = sharing =>
-  sharing.attributes.rules
+export const getSharingDocIds = sharing => {
+  const docs = sharing.attributes.rules
     .map(r => r.values)
     .reduce((acc, val) => acc.concat(val), [])
 
+  if (sharing.attributes.shortcut_id) {
+    docs.push(sharing.attributes.shortcut_id)
```

Comment:
Does that mean that the created shortcut is not shared itself?

Reply:
That's correct.

Reply:
OK so we create a fake shared docs. So if we go to the sharing views, we'll see it? 
Since the doc will be "shared", what happens if we open the Sharing Modal on it? Are we displaying "Create a link" or remove the link? And what happen if we click on that action? 
To be more precise, getSharingDocs is used to call `createSharingInStore` and then we add this document in the sharing store. I'm not sure of what happen next 

I'm not a big fan of creating fake shared document since that action can have a few consequences (error when trying to remove the sharing or so )

Reply:
> So if we go to the sharing views, we'll see it?

Yes, while the shortcut is there we're basically treating this file as if it was the real file and not the preview.

> what happens if we open the Sharing Modal on it?

We show the sharing infos for the actual sharing. Not sure if we have the full list of recipients before accepting the sharing, but at least the sharing owner and current user.

> Are we displaying "Create a link" or remove the link? And what happen if we click on that action?

Sharing by link is a totally separate mechanic. So… it's like a normal file, you can share the shortcut by link if you want.

> and then we add this document in the sharing store. I'm not sure of what happen next

Not much from what I can tell. It's in the `byDocId` slice of the store, which gives us an easy access to the related `io.cozy.sharings`.

______

We could not create "fake" shared docs, since in the short io.cozy.files metadata, we have the id of the sharing. But this would require handling this as a special case in drive in multiple places, as well a major changes in the cozy-sharings component so that they accept a `bySharingId` prop on top of `byDocId`.